packages/storybook/stories/Stateful Client/BestPractices.stories.mdx

```diff
@@ -92,4 +92,4 @@ Note that this only applies to remote screenshares and not local screenshares.
 ## Detecting Transcription and Recording
 
 Stateful Calling will manage transcription/recording for you.
-You only need to simply check the states under `CallState.transcription` and `CallState.recording`.
+You only need to simply check the states under `callstate.transcription` and `callstate.recording`.
```

Comment:
Do we consider them as one word or we use callState (I believe we use CallState as our type name, so they are considered to be 2 words)?

Reply:
Updated 👍